library/init/meta/attribute.lean

```diff
@@ -10,8 +10,16 @@ meta_constant attribute.get_instances : name → tactic (list name)
 meta_constant attribute.fingerprint : name → tactic nat
 
 structure user_attribute :=
+(name : name)
```

Comment:
The attribute name now comes directly from the structure instead of the definition's name so that `caching_user_attribute.get_cache` can access it.